### ISSUE TITLE
bilinear_interp输出shape为啥是[unk__34,unk__35,unk__36,unk__37] 我已经固定inputshape [1 3 512 512] 

### ISSUE THREAD

**Issue by 13950182204** [PaddlePaddle/Paddle2ONNX#1180]
**请将下面信息填写完整，便于我们快速解决问题，谢谢！**

**问题描述**
使用paddleseg2.9 版本 pp 2.6 版本 转换ppseg模型bilinear_interp输出shape 是[unk__34,unk__35,unk__36,unk__37] 
输出的shape也是[unk__34,unk__35,unk__36,unk__37] 

**更多信息 :**
我用于paddle2onnx 转onnx 再转rknn模型的
现在卡在paddle2onnx这个过程中
**报错截图**
![微信图片_20240126204855](https://github.com/PaddlePaddle/Paddle2ONNX/assets/102646780/101843fc-3dd0-42ce-88ff-bad17cf40669)

**其他信息**

**Comment by 13950182204** [comment]
batch_size: 64  #设定batch_size的值即为迭代一次送入网络的图片数量，一般显卡显存越大，batch_size的值可以越大。如果使用多卡训练，总得batch size等于该batch size乘以卡数。
iters: 10000    #模型训练迭代的轮数

train_dataset:  #训练数据设置
  type: Dataset #指定加载数据集的类。数据集类的代码在`PaddleSeg/paddleseg/datasets`目录下。
  dataset_root: /home/aistudio/data/carlane/
  train_path: /home/aistudio/data/carlane/train_list.txt  #数据集中用于训练的标识文件
  num_classes: 7  #指定类别个数（背景也算为一类）
  mode: train #表示用于训练
  transforms:
    - type: ResizeStepScaling
      min_scale_factor: 0.5
      max_scale_factor: 2.0
      scale_step_size: 0.25
    - type: RandomPaddingCrop
      crop_size: [512, 512]
    - type: RandomHorizontalFlip
    - type: RandomDistort
      brightness_range: 0.5
      contrast_range: 0.5
      saturation_range: 0.5
    - type: Normalize
  mode: train

val_dataset:  #验证数据设置
  type: Dataset #指定加载数据集的类。数据集类的代码在`PaddleSeg/paddleseg/datasets`目录下。
  dataset_root: /home/aistudio/data/carlane/ #数据集路径
  val_path: /home/aistudio/data/carlane/val_list.txt  #数据集中用于验证的标识文件
  num_classes: 7  #指定类别个数（背景也算为一类）
  mode: val #表示用于验证
  transforms: #模型验证的数据预处理的方式
    - type: Normalize #对原始图像进行归一化，标注图像保持不变

optimizer:
  type: AdamW
  weight_decay: 0.01

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.01
  end_lr: 0
  power: 0.9


loss:
  types:
    - type: OhemCrossEntropyLoss
      min_kept: 130000   # batch_size * 1024 * 512 // 16
    - type: OhemCrossEntropyLoss
      min_kept: 130000
    - type: OhemCrossEntropyLoss
      min_kept: 130000
  coef: [1, 1, 1]



model:
  type: PPLiteSeg
  num_classes: 7
  backbone:
    type: STDC1
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/PP_STDCNet1.tar.gz
  arm_out_chs: [32, 64, 128]
  seg_head_inter_chs: [32, 64, 64]
这个是 在paddleseg 上训练的yml

**Comment by 13950182204** [comment]
! python tools/export.py \
       --config  ../DeeplabV3.yml \
       --model_path output/iter_100/model.pdparams\
       --input_shape -1 3 512 512
!paddle2onnx --model_dir ./output/inference_model/ \
    --model_filename model.pdmodel \
    --params_filename model.pdiparams \
    --save_file segonnx/model.onnx \
    --opset_version 12 \
    --enable_dev_version True
!python -m paddle2onnx.optimize --input_model  segonnx/model.onnx \
                            --output_model  segonnx/PP_seg.onnx \
                            --input_shape_dict "{'x':[1,3,512,512]}"

**Comment by 13950182204** [comment]
![model (2)](https://github.com/PaddlePaddle/Paddle2ONNX/assets/102646780/a77bd86f-0381-4178-ab58-f7bb93aabcae)
这个是输出paddle 输出的模型也是固定的shape最后输出还是有问题

**Comment by 1314520gu** [comment]
请问解决了吗

**Comment by amandazw** [comment]
> batch_size: 64 #设定batch_size的值即为迭代一次送入网络的图片数量，一般显卡显存越大，batch_size的值可以越大。如果使用多卡训练，总得batch size等于该batch size乘以卡数。 iters: 10000 #模型训练迭代的轮数
> 
> train_dataset: #训练数据设置 type: Dataset #指定加载数据集的类。数据集类的代码在`PaddleSeg/paddleseg/datasets`目录下。 dataset_root: /home/aistudio/data/carlane/ train_path: /home/aistudio/data/carlane/train_list.txt #数据集中用于训练的标识文件 num_classes: 7 #指定类别个数（背景也算为一类） mode: train #表示用于训练 transforms: - type: ResizeStepScaling min_scale_factor: 0.5 max_scale_factor: 2.0 scale_step_size: 0.25 - type: RandomPaddingCrop crop_size: [512, 512] - type: RandomHorizontalFlip - type: RandomDistort brightness_range: 0.5 contrast_range: 0.5 saturation_range: 0.5 - type: Normalize mode: train
> 
> val_dataset: #验证数据设置 type: Dataset #指定加载数据集的类。数据集类的代码在`PaddleSeg/paddleseg/datasets`目录下。 dataset_root: /home/aistudio/data/carlane/ #数据集路径 val_path: /home/aistudio/data/carlane/val_list.txt #数据集中用于验证的标识文件 num_classes: 7 #指定类别个数（背景也算为一类） mode: val #表示用于验证 transforms: #模型验证的数据预处理的方式 - type: Normalize #对原始图像进行归一化，标注图像保持不变
> 
> optimizer: type: AdamW weight_decay: 0.01
> 
> lr_scheduler: type: PolynomialDecay learning_rate: 0.01 end_lr: 0 power: 0.9
> 
> loss: types: - type: OhemCrossEntropyLoss min_kept: 130000 # batch_size * 1024 * 512 // 16 - type: OhemCrossEntropyLoss min_kept: 130000 - type: OhemCrossEntropyLoss min_kept: 130000 coef: [1, 1, 1]
> 
> model: type: PPLiteSeg num_classes: 7 backbone: type: STDC1 pretrained: https://bj.bcebos.com/paddleseg/dygraph/PP_STDCNet1.tar.gz arm_out_chs: [32, 64, 128] seg_head_inter_chs: [32, 64, 64] 这个是 在paddleseg 上训练的yml

你这个提醒我了，我也遇到一样的问题，其实就是batch的问题，在转模型的时候把batch都改为1，我改完就没有这个错了，但是还没验证模型推理的正确性
